tools/fvm-bench/src/fevm.rs

```diff
@@ -37,11 +47,19 @@ pub fn run(
     let invoke_res = testkit::fevm::invoke_contract(tester, &mut account, actor, &input_data, gas)?;
 
     if !invoke_res.msg_receipt.exit_code.is_success() {
-        return Err(anyhow!(
-            "contract invocation failed: {} -- {:?}",
-            invoke_res.msg_receipt.exit_code,
-            invoke_res.failure_info,
-        ));
+        return Err(match invoke_res.failure_info {
```

Comment:
would printing traces not be useful here as well?

Reply:
We do below. But yeah, I should probably move things around.